### ISSUE TITLE
Move from SBCL

### ISSUE THREAD

**Issue by cottsay** [ros/roslisp#14]
This issue issue more or less intends to collect thoughts about if SBCL is the right CLISP implementation for use in ROS.

The primary motivation behind bringing this up is the fact that SBCL is not currently available on ARM, which is becoming more and more useful (and supported) in ROS. This makes `roslisp` the only `desktop-full` package which will not install on ARM.

How reasonable would it be to begin migrating to a more widely-available CLSIP implementation such as GNU CLISP? Would it even be possible? How much work?

Looking for thoughts...

**Comment by rohanbhargava11** [comment]
Any updates on this issue? I tried publishing to a topic using roslisp but I can't as sbcl doesn't support multi threading on ARM. I get this error 
 ; Evaluation aborted on #<SIMPLE-ERROR "Not supported in unithread builds." {52F5ED39}>.

Is there a workaround?

**Comment by cottsay** [comment]
SBCL is now supported on ARM. It is in the Fedora main repos, but I'm not sure about Ubuntu.

**Comment by rohanbhargava11** [comment]
Yes SBCL is supported in ubuntu as well. The issue here is whether multi threading is supported or not?  It looks like sbcl doesn't support multi threading on ARM. When I run the following command in sbcl running on arm 
(member :sb-thread *features*) I get nil. That looks like to me that multi threading is not supported therefore I can't run roslisp on ARM

**Comment by airballking** [comment]
No news on this from us in Bremen. We neither have the resources to investigate moving away from SBCL nor do we use ARM ourselves.

**Comment by gaya-** [comment]
According to the following commit, on ARM64 there is multithreading support now:
https://github.com/sbcl/sbcl/commit/53cbb34513ccf784bb34b42aafec59413f093e80
Can't test it myself though.

**Comment by gaya-** [comment]
SBCL has ARM support for 10 years now, so the problems that triggered this issue are solved.
